### PR TITLE
B 1682 mkp vote for project

### PR DESCRIPTION
**Pull request by haydencleary**

### CODE REVIEW

File: api-client/resources/me/queries/index.ts [hunk 1]
```diff
@@ -1 +1 @@
-// export queries
+export * from "api-client/resources/me/queries/use-get-my-committee-assignments";
```
Comment (alexbeno) [inline api-client/resources/me/queries/index.ts:1]:
```suggestion
export * from "./use-get-my-committee-assignments";
```

File: app/c/[committeeId]/jury/features/project-vote/project-vote.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+  // useMutationAlert({
+  //     mutation: restMutation,
+  //     success: {
+  //         message: T("v2.pages.committees.applicant.private.form.success"),
+  //     },
+  //     error: {
+  //         default: true,
+  //     },
+  // });
```
Comment (alexbeno) [inline app/c/[committeeId]/jury/features/project-vote/project-vote.tsx:23]: keep for later ?

File: app/c/[committeeId]/jury/features/projects-accordion/projects-accordion.tsx [hunk 3]
```diff
@@ -9,19 +10,24 @@ import { Avatar } from "components/ds/avatar/avatar";
...
+          title={<Typography variant="body-m-bold">{project.name}</Typography>}
           subtitle={
-            <Typography variant="body-s" className="text-spaceBlue-200">
-              {p.description}
-            </Typography>
+            // project.shortDescription ? (
+            //   <Typography variant="body-s" className="text-spaceBlue-200">
+            //     {project.shortDescription}
+            //   </Typography>
+            // ) : null
```
Comment (alexbeno) [inline app/c/[committeeId]/jury/features/projects-accordion/projects-accordion.tsx:29]: what we do with that ?

File: app/c/[committeeId]/jury/features/projects-accordion/projects-accordion.tsx [hunk 4]
```diff
@@ -47,7 +52,21 @@ export function ProjectsAccordion({ projects }: TProjectAccordion.Props) {
...
+              {
+                message: "Criteria 1",
+                score: 0,
+              },
+              {
+                message:
+                  "Criteria 2 hqsdfh qsdklfhqs klfqshfkl qshfkqs fqsklhf klsdhf qsklfhql kjhqsdfkqhsfklqs fklqs kflqsh fkjlqsdhfkqshfkqsjh fkqsh fqsfklqsh fkjlqsh dhkqsdh fkjqsdhfk qsd",
+                score: 3,
+              },
+            ]}
```
Comment (alexbeno) [inline app/c/[committeeId]/jury/features/projects-accordion/projects-accordion.tsx:68]: criteria can be fetcher thank to mock api

File: app/c/[committeeId]/jury/features/project-vote/project-vote.tsx (hunk 2 above)
Comment (haydencleary) [reply app/c/[committeeId]/jury/features/project-vote/project-vote.tsx:23]: yep

File: app/c/[committeeId]/jury/features/projects-accordion/projects-accordion.tsx (hunk 3 above)
Comment (haydencleary) [reply app/c/[committeeId]/jury/features/projects-accordion/projects-accordion.tsx:29]: keeping for later as the description isn't in the contract yet

File: app/c/[committeeId]/jury/features/projects-accordion/projects-accordion.tsx (hunk 4 above)
Comment (haydencleary) [reply app/c/[committeeId]/jury/features/projects-accordion/projects-accordion.tsx:68]: yep that will be in a future PR

**Review by alexbeno — APPROVED** [review]

**Pull request merged**